common/web/lm-worker/src/test/test-runner/cases/worker.spec.ts

```diff
@@ -1,25 +1,20 @@
-import { assert } from '../../../../../node_modules/chai/chai.js';
-
-import { Worker as WorkerBuilder }   from "../../../build/lib/web/index.mjs";
-import { LMLayerWorkerCode } from "/base/common/web/lm-worker/build/lib/worker-main.wrapped.min.js";
-import * as helpers from "../helpers.mjs";
+import { assert } from 'chai';
+import { LMLayerWorkerCode } from "@keymanapp/lm-worker/worker-main.wrapped.js";
 
 describe('LMLayerWorker', function () {
   // This one makes multiple subsequent calls across the WebWorker boundary, so we should be generous here.
-  this.timeout(Math.max(testconfig.timeouts.standard, 5000));
+  this.timeout(5000);
 
   describe('LMLayerWorkerCode', function() {
     it('should exist!', function() {
-      // assert.isFunction(LMLayerWorkerCode,
-      //   'Could not find LMLayerWorkerCode! Does embedded_worker.js exist?'
-      // );
       assert.isString(LMLayerWorkerCode);
     });
   });
 
   describe('Usage within a Web Worker', function () {
     it('should install itself in the worker context', function (done) {
-      let uri = WorkerBuilder.asBlobURI(LMLayerWorkerCode);
+      let blob = new Blob([LMLayerWorkerCode], { type: 'text/javascript' });
+      let uri = URL.createObjectURL(blob);
```

Comment:
This is a subtle change caused due to relocating the test; we shouldn't depend on the parent module's classes for a child module's test.